js_modules/dagit/packages/core/src/workspace/WorkspaceContext.tsx

```diff
@@ -1,4 +1,5 @@
 import {ApolloQueryResult, gql, useQuery} from '@apollo/client';
+import {sortBy} from 'lodash';
```

Comment:
I'm not sure how effectively we're tree-shaking `lodash` and we should probably investigate. Might be worth just doing this for now:

```
import sortBy from 'lodash/sortBy';
```

Reply:
Oh good call will swap that out